tests/test_fsspec.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+pytest.importorskip("moto")
```

Comment:
I'm happy to make `moto` a dev-dependency, so you can take this line out.

Reply:
I already is, but I didn't realise that :)